neo/SmartContract/StandardService.cs

```diff
@@ -21,7 +21,7 @@ public class StandardService : IDisposable, IInteropService
         public static event EventHandler<NotifyEventArgs> Notify;
         public static event EventHandler<LogEventArgs> Log;
 
-        protected readonly TriggerType Trigger;
+        public readonly TriggerType Trigger;
```

Comment:
Let's keep this protect and add a public `get`, it's better.

Reply:
For a readonly field, what's the difference?